src/App.js

```diff
@@ -16,53 +15,71 @@ class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
-      movie: ''
+      movies: [],
+      movie: '',
+      error: false,
+      videos: [],
     }
   }
 
-  addMovies = () => {
-    this.setState({ movies: [...movieData.movies] })
+  componentDidMount = () => {
+    fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
+    .then(response => response.json())
+    .then(data => this.setState({movies : data.movies}))
+    .catch(err => console.log(err))
```

Comment:
We will need to do something with this catch instead of just a console.log... I just threw it in there so that we would remember